config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -86,6 +86,59 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-node-e2e-kubetest2
+    # explicitly needs /test pull-kubernetes-node-e2e to run
+    always_run: false
+    # if it is run and fails, don't block the PR
+    optional: true
+    branches:
+    - master
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    decoration_config:
+      timeout: 65m
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-k8s-ssh: "true"
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210915-5dbaf53458-master
+        resources:
+          limits:
+            cpu: 4
+            memory: 6Gi
+          requests:
+            cpu: 4
+            memory: 6Gi
+        command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - kubetest2
+        - noop
+        - --build
+        - --up
+        - --down
+        - --test=node
+        - --
+        - --zone=us-west1-b
+        - --parallelism=8
+        - --focus-regex='\[NodeConformance\]'
+        - --skip-regex='\[Flaky\]|\[Slow\]|\[Serial\]'
+        - --test-args='--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"'
+        - --image-config-file='/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/image-config.yaml'
```

for now this is good to start with, once this job becomes stable let's add functionality in the node tester to be able to read files directly by specifying a raw url e.g. https://raw.githubusercontent.com/kubernetes/test-infra/master/jobs/e2e_node/image-config.yaml

---

I generally don't like seeing prow's paths hardcoded in job configs, but this isn't a blocking suggestion
```suggestion
        - --image-config-file='../test-infra/jobs/e2e_node/image-config.yaml'
```